jekyll/_cci2/openid-connect-tokens.adoc

```diff
@@ -141,20 +142,146 @@ jobs:
+
+=== Google Cloud Provider
+
+The following GCP instructions are for:
+
+* A one-time configuration of your GCP settings to trust CircleCI's OIDC tokens
+* Running a job that uses the OIDC token to interact with GCP
+
+The Google Cloud CLI reads your configuration file which contains necessary information instructing Google Cloud to authenticate. You can read about external identity providers on https://cloud.google.com/iam/docs/configuring-workload-identity-federation#oidc[Google Cloud's docs].
+
+==== Setting up GCP
+
+The GCP configuration file can be set up using the GCP UI. In the **Workload Identity Federation UI**, navigate to **Grant Access**, which will prompt the configuration, which can then be downloaded. You will need to create a file named `CIRCLE_OIDC_TOKEN_FILE`, which Google Cloud will read your identity token from.
+
+You will need your CircleCI organization ID, which can be found by navigating to **Organization Settings > Overview** on the https://app.circleci.com/[CircleCI web app].
+
+After navigating to the **Grant Access** section of the GCP UI, follow these steps to add CircleCI as an external identity provider:
+
+1. Navigate to the **IAM & Admin panel**
+1. On the side panel, navigate to **Workload Identity Federation**
+1. Click the **Add Provider** button
+1. Select **OpenID Connect (OIDC)** from the "Select a provider" dropdown and click **Save**
+1. Fill out the **Provider details** form
+  * Select **Allowed audiences** since the `aud` claim in the JSON Web Token is a UUID (your CircleCI organization ID). The `audience` will be your CircleCI organization ID
+  * The issuer is `\https://oidc.circleci.com/org/ORG_ID`, where `ORG_ID` is your CircleCI organization ID
+1. Click **Continue** to configure provider attributes
++
+Configuring the provider attributes provides an opportunity to map claims in CircleCI's Token to Google's "understanding." For example:
++
+[.table.table-striped]
+[cols=2*, stripes=even]
+
+|===
+| google.subject
+| attribute.project_id
+
+| assertion.sub
+| assertion['oidc.circleci.com/project-id']
+|===
++
+1. Navigate to **Service Account** in the IAM & Admin Panel to create a service account, and give appropriate permission
+1. Navigate back to **Workload Identity Federation** and select the provider from the table
+1. Click the **Grant access** button
+1. A modal will open and you will select the service account you created from the dropdown. This is the account that the token will impersonate, which grants all the associated permissions
+1. Under **Select principals**, you can add conditions, or leave the default
+1. Click **Save**. A pop-up will appear to ask you configure and **download** the configuration file. This file can also be downloaded later by navigating to **Connected Service Accounts**
+
```

Comment:
numbering looks off here too?

Reply:
@thomasr8 This is how auto-numbering works in asciidoc. If you label everything as 1, the numbers render in order. This makes it easy to update or change instructions without worrying about having to re-order or update numbers in the future.